doc/api/http2.md

```diff
@@ -2711,8 +2711,8 @@ added: v8.4.0
 This object is created internally by an HTTP server — not by the user. It is
 passed as the second parameter to the [`'request'`][] event.
 
-The response inherits from [Stream][], with some methods like what we have in
-[Writable Stream][]. It will emit the following events:
+The response inherits from [Stream][], with two methods like what we have in
+[Writable Stream][]: [`response.write()`][] and [`response.end()`][].
```

Comment:
Is `It will emit the following events:` omission intentional?

Reply:
I don't think we usually preface documented events with a sentence like that so I'm for removing it.